quota/provider.go

```diff
@@ -78,13 +64,13 @@ func quotaSystems() []string {
 	return r
 }
 
-// NewQuotaManagerFromFlags returns a quota.Manager implementation as speficied by flag.
-func NewQuotaManagerFromFlags() (quota.Manager, error) {
-	return NewQuotaManager(*QuotaSystem)
+// NewManagerFromFlags returns a Manager implementation as speficied by flag.
+func NewManagerFromFlags() (Manager, error) {
+	return NewManager(*System)
 }
 
-// NewQuotaManager returns a quota.Manager implementation.
-func NewQuotaManager(name string) (quota.Manager, error) {
+// NewManager returns a Manager implementation.
+func NewManager(name string) (Manager, error) {
```

Comment:
For consistency, consider renaming `Manager` bits to `Provider` throughout this PR.

Reply:
A `Manager` is actually akin to a storage implementation - it's not a provider. In this case, `NewManagerFunc` is the provider. I could rename that to `ProviderFunc` if you think that'd be an improvement?